pkgs/by-name/ba/bant/package.nix

```diff
@@ -4,8 +4,7 @@
   buildBazelPackage,
   fetchFromGitHub,
   bazel_6,
-  jdk,
-  git,
+  jdk
```

Comment:
I'd always keep the comma at the end, it makes it easier to extend in the future and to re-organize imports.